recipes/polap-fmlrc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "polap-fmlrc" %}
+{% set version = "0.2.4" %}
+{% set sha256 = "4463d4a71a8a3273108a0ab2dff0a4c37a565c05af6ee17ee83dc62404294caf" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/goshng/polap/archive/refs/tags/{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('polap-fmlrc', max_pin="x.x") }}
+
+requirements:
+  run:
+    - python =2.7
+    - bioconda::ropebwt2 >=r187
```

Comment:
Please do not specify channels here

Reply:
Let me see ... so I should not append the channel name to a bioconda and conda-forge package. Right?

Reply:
I am editing the recipe.

Reply:
I have corrected them. Thank you.

Reply:
Yes

Reply:
Thank you